apm-sniffer/apm-sdk-plugin/armeria-0.8.x-plugin/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>apm-sdk-plugin</artifactId>
+        <groupId>org.apache.skywalking</groupId>
+        <version>6.6.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>apm-armeria-0.8.x-plugin</artifactId>
+    <name>armeria-0.8.x-plugin</name>
+    <packaging>jar</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.linecorp.armeria</groupId>
+            <artifactId>armeria</artifactId>
+            <version>0.84.0</version>
```

Comment:
Could you use [0.97.0](https://search.maven.org/artifact/com.linecorp.armeria/armeria-bom/0.97.0/pom) which is the latest version of Armeria release before proceeding further? 😉

Reply:
@kezhenxu94 Besides compile version, Seems there are more new versions, please choose some of them to run the tests, but no may not need all of them. Your choice.

Reply:
> Could you use [0.97.0](https://search.maven.org/artifact/com.linecorp.armeria/armeria-bom/0.97.0/pom) which is the latest version of Armeria release before proceeding further? 😉

Huh, that's really new and hot, was just released hours ago, let me give it a try